packages/ckb-sdk-core/examples/sendTransaction.js

```diff
@@ -35,7 +46,7 @@ const bootstrap = async () => {
   const lockScript = {
     hashType: "type",
     codeHash: blockAssemblerCodeHash,
-    args: [myAddressObj.publicKeyHash],
+    args: [publicKeyHash],
```

Comment:
now args is a Bytes not an array?

Reply:
> now args is a Bytes not an array?

fixed in https://github.com/nervosnetwork/ckb-sdk-js/pull/369/commits/9bcec6b2a79001710e5ec74819d2efd8011210b0